ava-labs/core-extension
# feat: add X/P private key export CP-9337

## Description

Adds capability to export private keys for X/P chain.
Fixes PK export when exporting on a non active account

## Changes

- Adds dropdown for chain selection for  mnemonic PK exports

## Testing

- Export PK for C and X/P  from a mnemonic wallet when it's the active primary wallet. Verify it's working correctly.
- Export PK for C and X/P from a mnemonic wallet when it's NOT the active primary wallet. Make sure a NON Mnemonic wallet is active. Verify it's working correctly.
- Export PK for C and X/P from a mnemonic wallet when it's NOT the active primary wallet. Make sure an imported account is active. Verify it's working correctly.

## Screenshots:

![Screenshot 2024-10-10 at 15 13 53](https://github.com/user-attachments/assets/b3c080b6-76bf-48c5-94fb-11f577e4aa60

## Checklist for the author

Tick each of them when done or if not applicable.
